facebookresearch/droidlet
# Fix submodule version to stable, working ones as in main branch

# Description

The commits of submodules in this branch are different from those on main, causing some compatibility issues such as cuberite binary path change. Here fix the commits to be the same as those on main branch.


It also seems to solve the issue with shutil copy symlink issue in this commit: https://github.com/facebookresearch/droidlet/commit/cf06fc7537441849442f1d83b4dadfc7f8f1ef5c. @kavyasrinet if I revert this commit and use the updated submodule commits, it will work.

Please check the options that are relevant.

- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

Before:

cuberite_process.py can not be run -- failed to find Cuberite executable .

After:

cuberite_process.py can be run successfully

# Testing

Tested on devfair.

**Steps to test**

Since we are updating submodule versions, you should do this before test:

```
git submodule deinit -f .
git submodule update --init --recursive
```

Once you see no chanegs need to be staged using `git status`, run the following commands under root:
```
cd droidlet/lowlevel/minecraft
make cuberite
python cuberite_process.py
```

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I have added relevant collaborators to review the PR before merge.
